### DISCUSSION TITLE
Any plan to support SPM binaryTarget

### DISCUSSION THREAD

**Discussion by bluepi0j** [krzyzanowskim/CryptoSwift#1006] [General]
Do you have any plan to support SPM binaryTarget. Like what https://github.com/airbnb/lottie-spm Lottie did (a separate repo)  or Facebook sdk https://github.com/facebook/facebook-ios-sdk/blob/main/Package.swift do. It can download quicker and reduce build time

**Comment by krzyzanowskim** [comment]
is Xcode ready? I did that once, and reverted due to many complains that Xcode can't handle frameworks properly
